client/api_client.go

```diff
@@ -60,6 +60,8 @@ type ApiClientInterface interface {
 	EnvironmentScheduling(environmentId string) (EnvironmentScheduling, error)
 	EnvironmentSchedulingUpdate(environmentId string, payload EnvironmentScheduling) (EnvironmentScheduling, error)
 	EnvironmentSchedulingDelete(environmentId string) error
+	WorkflowTrigger(environmentId string) ([]WorkflowTrigger, error)
+	WorkflowTriggerCreate(environmentId string, request WorkflowTriggerCreatePayload) (WorkflowTrigger, error)
```

Comment:
It's Update, not Create
Or Upsert

Reply:
b0c6ca74(upsert + type)
also i was wrong on the return type